src/esm_parser/esm_parser.py

```diff
@@ -1468,9 +1468,21 @@ def resolve_basic_choose(config, config_to_replace_in, choose_key, blackdict={})
         choice = do_math_in_entry([False], choice, config)
     logging.debug(choice)
 
-    if choice in config_to_replace_in.get(choose_key):
+    # This allows users to use version numbers (floats) and integers as choices inside
+    # the choose_blocks, instead of having to specify the choices as strings
+    if isinstance(choice, (int, float)) and not isinstance(choice, bool):
+        choice = str(choice)
+    choices_available = {}
+    for ckey, cval in config_to_replace_in.get(choose_key, {}).items():
+        if isinstance(ckey, (int, float)) and not isinstance(ckey, bool):
+            choices_available[str(ckey)] = cval
+        else:
+            choices_available[ckey] = cval
+
+    # Resolve the choose variables
+    if choice in choices_available:
         for update_key, update_value in six.iteritems(
```

Comment:
Cleanup: we need to get rid of six at some point. It doesn't hurt, of course, but one more thing that is divergent everywhere in the code.

 I think I will ask Natalja is I can rent a cabin in northern Sweden for a week and just tear apart everything to do the basic cleaning

Reply:
Can it be a cabin for 2 + cat?

Reply:
Well, then of course for all the esm team. But I am sure we need to first write a 5 page document outlining out covid safety concept.